dist/gl-matrix.js

```diff
@@ -1697,51 +1697,53 @@ return /******/ (function(modules) { // webpackBootstrap
 	 * @returns {mat4} out
 	 */
 	mat4.SIMD.multiply = function (out, a, b) {
-	    var _a = SIMD.Float32x4.load(b, 0);
-	    var _b = SIMD.Float32x4.load(b, 4);
-	    var _c = SIMD.Float32x4.load(b, 8);
-	    var _d = SIMD.Float32x4.load(b, 12);
-
-	    var t1 = SIMD.Float32x4.splat(a[0]);
-	    var t2 = SIMD.Float32x4.mul(_a, t1);
-	    t1 = SIMD.Float32x4.splat(a[1]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_b, t1), t2);
-	    t1 = SIMD.Float32x4.splat(a[2]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_c, t1), t2);
-	    t1 = SIMD.Float32x4.splat(a[3]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_d, t1), t2);
-	    SIMD.Float32x4.store(out, 0, t2);
-
-	    t1 = SIMD.Float32x4.splat(a[4]);
-	    t2 = SIMD.Float32x4.mul(_a, t1);
-	    t1 = SIMD.Float32x4.splat(a[5]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_b, t1), t2);
-	    t1 = SIMD.Float32x4.splat(a[6]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_c, t1), t2);
-	    t1 = SIMD.Float32x4.splat(a[7]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_d, t1), t2);
-	    SIMD.Float32x4.store(out, 4, t2);
-
-	    t1 = SIMD.Float32x4.splat(a[8]);
-	    t2 = SIMD.Float32x4.mul(_a, t1);
-	    t1 = SIMD.Float32x4.splat(a[9]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_b, t1), t2);
-	    t1 = SIMD.Float32x4.splat(a[10]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_c, t1), t2);
-	    t1 = SIMD.Float32x4.splat(a[11]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_d, t1), t2);
-	    SIMD.Float32x4.store(out, 8, t2);
-
-	    t1 = SIMD.Float32x4.splat(a[12]);
-	    t2 = SIMD.Float32x4.mul(_a, t1);
-	    t1 = SIMD.Float32x4.splat(a[13]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_b, t1), t2);
-	    t1 = SIMD.Float32x4.splat(a[14]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_c, t1), t2);
-	    t1 = SIMD.Float32x4.splat(a[15]);
-	    t2 = SIMD.Float32x4.add(SIMD.Float32x4.mul(_d, t1), t2);
-	    SIMD.Float32x4.store(out, 12, t2);
-
+	    var a0 = SIMD.Float32x4.load(a, 0);
+	    var a1 = SIMD.Float32x4.load(a, 4);
+	    var a2 = SIMD.Float32x4.load(a, 8);
+	    var a3 = SIMD.Float32x4.load(a, 12);
+
+	    var b0 = SIMD.Float32x4.load(b, 0);
+	    var b1 = SIMD.Float32x4.load(b, 4);
```

Comment:
Yeah, did I mention that Jukka is a time traveler sent from the future to save us?